src/cq_cam/routers.py

```diff
@@ -211,23 +213,24 @@ def route_wires(job: "Job", wires: List[Union[cq.Wire, cq.Edge]], stepover=None)
         else:
             distance, target, param = None, None, None
 
-        if (
-            ep
-            and isclose(ep.x, start.x, abs_tol=0.001)
-            and isclose(ep.y, start.y, abs_tol=0.001)
-        ):
-            commands.append(Plunge.abs(start.z, arrow=True, feed=job.feed))
-        elif stepover and distance and distance <= stepover:
+        # This check is only relevant for the pocket operation as it is the only one that sets the stepover.
```

Comment:
Technically the pocket uses now as default clipper so route_polyface_outers instead of route_wires